src/sagas/import-project.saga.test.js

```diff
@@ -169,6 +175,10 @@ describe('import-project saga', () => {
           createdAt: 1532809641976,
         },
       };
+
+      const getState = window.store.getState();
+      const onboardingCompleted = getState.onboardingStatus === 'done';
```

Comment:
While it's less critical in tests, I think we should also avoid using `window.store` here.

In fact, for the purposes of this test, it's actually better if we hardcode it:

```js
expect(saga.next(jsonWithGuppy).value).toEqual(
  put(
          importExistingProjectFinish(
            'path/to/project',
            jsonWithGuppy,
            true // Use a hardcoded value
          )
  )
)
```

This way we can make sure the saga is passing along the right value, and not just whatever value it generates internally.